plugins/confluence-plugin/src/components/PageContent.test.tsx

```diff
@@ -0,0 +1,101 @@
+      representation: "view",
+      _expandable: {
+        webresource: "",
+        embeddedContent: "",
+        mediaToken: "",
+        content: "/rest/api/content/131073",
+      },
+    },
+    _expandable: {
+      editor: "",
+      atlas_doc_format: "",
+      export_view: "",
+      styled_view: "",
+      dynamic: "",
+      storage: "",
+      editor2: "",
+      anonymous_export_view: "",
+    },
+  },
+};
+
+const serviceYaml = {
+  info: {
+    "x-cortex-confluence": { pageID: 131073 },
+  },
+};
+
+describe("PageContent", () => {
+  beforeEach(() => {
+    fetchMock.resetMocks();
+  });
+
+  it("shows message when no page is found", async () => {
+    fetchMock.mockResponses(
+      [JSON.stringify(serviceYaml), { status: 200 }],
+      [JSON.stringify({}), { status: 500 }]
+    );
+
+    render(<PageContent />);
+
+    await waitFor(() => {
+      expect(
+        screen.getByText(
+          "We could not find any Confluence Page associated with this entity"
+        )
+      ).toBeInTheDocument();
+    });
+  });
+
+  it("Shows a page if page is found", async () => {
```

Comment:
nit: these should read and be capitalized like sentences starting with "it"
```suggestion
  it("shows a page if page is found", async () => {
```